src/Gui/ViewProviderSuppressibleExtension.h

```diff
@@ -0,0 +1,54 @@
+/***************************************************************************
+ *   Copyright (c) 2024 Florian Foinant-Willig <ffw@2f2v.fr>               *
+ *                                                                         *
+ *   This file is part of the FreeCAD CAx development system.              *
+ *                                                                         *
+ *   This library is free software; you can redistribute it and/or         *
+ *   modify it under the terms of the GNU Library General Public           *
+ *   License as published by the Free Software Foundation; either          *
+ *   version 2 of the License, or (at your option) any later version.      *
+ *                                                                         *
+ *   This library  is distributed in the hope that it will be useful,      *
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+ *   GNU Library General Public License for more details.                  *
+ *                                                                         *
+ *   You should have received a copy of the GNU Library General Public     *
+ *   License along with this library; see the file COPYING.LIB. If not,    *
+ *   write to the Free Software Foundation, Inc., 59 Temple Place,         *
+ *   Suite 330, Boston, MA  02111-1307, USA                                *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef VIEWPROVIDERSUPPRESSIBLEEXTENSION_H
+#define VIEWPROVIDERSUPPRESSIBLEEXTENSION_H
+
+#include "ViewProviderExtensionPython.h"
+
+
+namespace Gui
+{
+
+class ViewProviderSuppressibleExtension : public Gui::ViewProviderExtension
```

Comment:
This line should be : 
`class GuiExport ViewProviderSuppressibleExtension : public ViewProviderExtension`